core/tests/protractor/libraryFlow.js

```diff
@@ -136,6 +136,16 @@ describe('Rating', function() {
      'submitted', async function() {
     await users.createAndLoginSuperAdminUser(
       'user11@explorationRating.com', 'user11Rating');
+
+    // The below lines of code enable the user checkpoints feature on the
+    // config tab. This is required to enable the lesson-info modal button
+    // on the exploration footer, which in turn is required to view the ratings.
+    // This should be removed when the user checkpoints feature is no longer
+    // gated behind a config option.
+    await adminPage.editConfigProperty(
+      'Enable checkpoints feature.', 'Boolean',
+      async(elem) => await elem.setValue(true));
```

Comment:
Can you wait to make sure the element has loaded before you try and set its value?

Reply:
Hi @U8NWXD, actually, the function responsible for saving a config [does wait for the element](https://github.com/oppia/oppia/blob/8f04c4b559eab93f28c155606106e9e86501cb5f/core/tests/protractor_utils/AdminPage.js#L151), so I couldn't quite understand what you meant. I've added an await for the element whose value is being set, please lmk if this is what you meant.

Reply:
Ahh I missed the `waitFor.visibilityOf` here:

https://github.com/oppia/oppia/blob/8f04c4b559eab93f28c155606106e9e86501cb5f/core/tests/protractor_utils/AdminPage.js#L145

LGTM!